### DISCUSSION TITLE
Vaccination

### DISCUSSION THREAD

**Discussion by DiegoCorchuelo** [Launch-X-Latam/MisionFrontEnd#377] [Show and tell]
Hola compas, he terminado la practica y me gustaría recibir un poco de feedback.  

https://diegocorchuelo.github.io/Vaccination/

**Comment by TonatiuhDSJ** [comment]
Esta super genial, solo que si me sigo para abajo, tiene un espacio en blanco. 
no se si sea el footer o algo, fuera de ello ni yo podria haberlo hecho

**Reply by DiegoCorchuelo** [reply]
Muchisimas gracias <3.  
Podrias enviarme un screenshot de eso please.

**Reply by TonatiuhDSJ** [reply]
no como crees, gracias a ti.
Y claro aqui esta.
![Screenshot 2022-11-09 161758](https://user-images.githubusercontent.com/39995859/200954236-2a42cad4-6968-4de5-93fe-f1f0205397f7.png)

**Reply by DiegoCorchuelo** [reply]
Wow, no se, supongo que será por que no tiene responsive. 
Tú tambien dale que puedes hacerlo y mejor que yo.
Gracias por el feedback <3

**Reply by TonatiuhDSJ** [reply]
Muchas gracias por tu apoyo y en una de esas si es el responsive.
Ojala sea cosa de nada.
Gracias por tu comentario :D

**Comment by OscarJG20** [comment]
Hola!
En general la parte visual se ve muy bien y parecido a la imagen que se nos brindo, sin embargo en la versión escritorio ya tiene un desbordamiento (scroll horizontal), creo que lo deberías solucionar antes, si piensas hacerlo responsive, porque sino te va a dar un dolor de cabeza tremendo, ánimo!

**Reply by DiegoCorchuelo** [reply]
Muchas Gracias Oscar, Te cuento que tengo una pantalla larga y a mi no me sale el scroll horizontal.
Supongo que es por el responsive pero si tienes alguna otra idea que pueda  ayudarme me encantaría escucharla.
